src/shared/copy/notifications/categories/accounts-users-orgs.ts

```diff
@@ -4,6 +4,11 @@ import {
   defaultSuccessNotification,
 } from 'src/shared/copy/notifications'
 
+export const defaultAccountNotFoundError = (): Notification => ({
```

Comment:
I don't think this notification that's been added is being used anywhere, but please LMK if that's wrong.

Reply:
ah yea, i added this and then decided to go against it. will add it in the above mentioned PR.

Reply:
removed in the other PR.